app/scripts/constants.js

```diff
@@ -687,5 +687,10 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     'openshift-',
     'kubernetes-',
     'kube-'
-  ]
+  ],
+  MOBILE_CLIENT_VERSION: {
+    group: 'mobile.k8s.io',
+    version: 'v1alpha1',
+    resource: 'mobileclients'
```

Comment:
This should probably go in origin-web-common preferred versions.

Reply:
@spadgett I`ll update once this gets approved https://github.com/openshift/origin-web-common/pull/320

Reply:
Yikes, sorry that has sat so long :( Looks like that is right around when 3.10 feature freeze happened. We'll get that integrated as soon as we branch for 3.11.

Reply:
no worries at all, fantastic, thank you